key.js

```diff
@@ -1,18 +1,261 @@
+function animateToLevelZero() {
+    defaultLife.textContent = `  x ${gameState.lives}`;
+    heart.style.display = ``;
+    timerElement.style.display = ``;
+    startButton.style.fontSize = `45px`
+    timer = setInterval(() => {
+        gameState.seconds = gameState.seconds - 1;
+        timerElement.textContent = `${gameState.seconds} `;
+        if (gameState.seconds <= 0) {
+            clearInterval(timer);
+        }
+    }, 1000)
+    timerElement.style.boxShadow = `0px 0px 4px 3px darkgray`;
+    startButton.style.transform = ``
+    h3.style.display = ``;
+    keyBoard.style.display = ``;
+    startButton.textContent = `HOME`;
+    startButton.style.transform = `translateX(450%) translateY(-250px)`
+    startButton.style.backgroundColor = `red`;
+    startButton.style.color = `white`;
+    document.body.style.backgroundColor = `whitesmoke`;
+    startButton.style.border = `5px solid darkred`
+    headings.textContent = `Baby Level`;
+    headings.style.color = "darkslategray";
+}
+
+function animateToHomePage() {
```

Comment:
ditto

Reply:
I don't understand ditto, tell me if it's going in the controller module or not?